runtime/amd64.S

```diff
@@ -735,6 +735,7 @@ LBL(caml_c_call):
     /* Save non-callee-saved registers %rax and %xmm0 before C call */
         pushq   %rax; CFI_ADJUST(8);
         subq    $16, %rsp; CFI_ADJUST(16);
+    /* Save %xmm0 before C call */
```

Comment:
This line looks a bit redundant with the comment just above.

Reply:
Fixed in latest version (I also squashed the history)